runtime/gc_base/IndexableObjectAllocationModel.cpp

```diff
@@ -133,24 +158,57 @@ MM_IndexableObjectAllocationModel::initializeIndexableObject(MM_EnvironmentBase
 #endif /* defined(J9VM_ENV_DATA64) */
 		} else {
 			indexableObjectModel->setSizeInElementsForContiguous(spine, _numberOfIndexedFields);
+			isArrayletDataAdjacentToHeader = indexableObjectModel->isArrayletDataAdjacentToHeader(spine);
+			if (isArrayletDataAdjacentToHeader) {
+				getAllocateDescription()->setDataAdjacentToHeader(true);
```

Comment:
We cache this value in Allocate Description but seems never use it. And `_dataAdjacentToHeader` has not been used before. Do we need this variable at all, is there any value? If we don't need this variable we can remove it from Allocate Description (can be done in separate PR)

Reply:
As this was a simple change, I have removed this now.